velox/common/memory/HashStringAllocator.cpp

```diff
@@ -616,4 +616,20 @@ void HashStringAllocator::checkConsistency() const {
   VELOX_CHECK_EQ(bytesInFreeList, freeBytes_);
 }
 
+void HashStringAllocator::checkEmpty() const {
+  for (auto i = 0; i < kNumFreeLists - 1; ++i) {
+    VELOX_CHECK(free_[i].empty());
+  }
+  auto numRanges = pool_.numRanges();
+  int32_t numFree = 0;
+  for (auto* item = free_[kNumFreeLists - 1].next();
```

Comment:
nit: consider extracting temp variable for free_[kNumFreeLists - 1] to improve readability